tap_dynamodb/sync_strategies/log_based.py

```diff
@@ -124,9 +145,25 @@ def sync(config, state, stream):
 
     md_map = metadata.to_map(stream['metadata'])
     projection = metadata.get(md_map, (), 'tap-mongodb.projection')
+    expression = metadata.get(md_map, (), 'tap-dynamodb.expression')
     if projection is not None:
         projection = [x.strip().split('.') for x in projection.split(',')]
-
+        if expression:
+            # decode the expression in jsonified object.
+            expression = dynamodb.decode_expression(expression)
+            # loop over the expression keys
+            for expr in expression.keys():
+                # check if the expression key starts with `#` and if not raise an exception
+                if not expr[0].startswith("#"):
+                    raise Exception("Expression key '{}' must start with '#'.".format(expr))
+
+            exp_key_traverse = set(expression.keys())
```

Comment:
Please provide the comment to define this variable

Reply:
Added comment